mitxgraders/sampling.py

```diff
@@ -668,8 +670,14 @@ def gen_symbols_samples(symbols, samples, sample_from, functions, suffixes, cons
         sample_list.append(sample_dict)
     return sample_list
 
+schema_user_functions_no_random = All(
```

Comment:
Can we insert a comment explaining this?

Reply:
added comment about NumericalGrader